tests/core/test_validation_definition.py

```diff
@@ -189,6 +189,9 @@ def test_returns_expected_data(
                 "unsuccessful_expectations": 0,
                 "success_percent": 100.0,
             },
+            meta={
+                "validation_id": None,
```

Comment:
Can the validation_id be None? In the text_checkpoint test above you verify that it's a UUID.

Reply:
It will be populated if and only if we have persisted the object before running. Do we need to update the checkpoint/validation_definition to persist on run if it doesn't already exist in the store?